services/npms-io/npms-io-score.service.js

```diff
@@ -23,7 +23,7 @@ export default class NpmsIOScore extends BaseJsonService {
   static route = {
     base: 'npms-io',
     pattern:
-      ':type(final|maintenance|popularity|quality)-score/:scope(@.+)?/:packageName',
+      'score/:type(final|maintenance|popularity|quality)/:scope(@.+)?/:packageName',
```

Comment:
Suppose this becomes a question of end user experience on the badge builder vs. processing of a request, but did you consider keeping the route the same and add `-score` duplicatively on each type variant? Recognize that'd be a bit verbose inline here and add a dash of noise in the modal, but it would obviate the need for a redirect, and arguably adheres closer to that single-segment noun under our badge route guidelines

Reply:
Weirdly, I actually hadn't thought of that, although it seems obvious now. In the interests of not introducing more redirects where we don't need them, I think I do prefer that option.
I've submitted https://github.com/badges/shields/pull/8967 which implements that suggestion.